src/components/Header/Pages.js

```diff
@@ -1,6 +1,6 @@
 import strings from 'lang';
```

Comment:
The NavBar component is actually not in use at this time.  You could remove it completely and inline the pages in the Header.